docs/topics/inline-classes.md

```diff
@@ -135,10 +153,26 @@ fun compute(x: UInt) { }
 To mitigate such issues, functions using inline classes are *mangled* by adding some stable hashcode to the function name. 
 Therefore, `fun compute(x: UInt)` will be represented as `public final void compute-<hashcode>(int x)`, which solves the clash problem.
 
-> `-` is an *invalid* symbol in Java, meaning that it's not possible to call functions which accept inline classes from Java.
+> Mangling schemes in Kotlin 1.4 and 1.4.30 are different. 
+> Use the `-Xuse-14-inline-classes-mangling-scheme` compiler flag to force the compiler to use the old 1.4.0 mangling scheme and preserve binary compatibility.
 >
 {type="note"}
 
+### Calling from Java code
+
+You can call functions which accept inline classes from Java code. To do that, you should disable mangling manually:
+add the `@JvmName` annotation before the function declaration:
+
+```kotlin
+@JvmInline
+value class UInt(val x: Int)
```

Comment:
Not sure it's a good example since there is a built-in class with the same name: https://kotlinlang.org/docs/basic-types.html#unsigned-integers
Please check.

Reply:
It also was in the previous doc version.

@ilmirus should we change the example?

Reply:
I suspect it was written before the unsigned types appeared :) 🦣

Reply:
So, we should also renew the example in KEEP https://github.com/Kotlin/KEEP/blob/master/proposals/inline-classes.md

Reply:
The class is not built-in, but rather standard. Given that it is implemented as an inline class, it is a perfect example of inline class, with JetBrains seal of approval.

So, I see no reason to change the example.